src/sgame/sg_admin.cpp

```diff
@@ -863,11 +863,15 @@ static bool admin_higher( gentity_t *admin, gentity_t *victim )
 	                           victim->client->pers.admin );
 }
 
-static void admin_writeconfig_string( char *s, fileHandle_t f )
+static void admin_writeconfig_string( Str::StringRef s, fileHandle_t f )
 {
-	if ( s[ 0 ] )
+	if ( !s.empty() )
 	{
-		trap_FS_Write( s, strlen( s ), f );
+		if ( ( s.front() & 0xFF ) <= ' ' || ( s.back() & 0xFF ) <= ' ' )
```

Comment:
Should you just use something like `!iscntrl()` or `isprint()`?

Reply:
The standard library functions are all unusable since they are supposedly locale-dependent (hence `Str::cislower` etc.).

I did this since it's the same thing that `SkipWhitespace` does

Reply:
Got it. Maybe leave a comment about  what is going on here.

Reply:
Done.